ngraph/core/include/openvino/runtime/tensor.hpp

```diff
@@ -50,10 +50,12 @@ class OPENVINO_API Tensor {
     friend class ov::runtime::VariableState;
 
 public:
+    using Ptr = std::shared_ptr<Tensor>;
     /**
      * @brief Default constructor
      */
     Tensor() = default;
+    virtual ~Tensor() = default;
```

Comment:
What is the purpose of virtual destructor?

Reply:
I need to have a virtual destructor, because I use internal class to wrap HostTensor to ov::runtime::Tensor.
And Tensor is a base class for the internal wrapper.
And I work with this class through pointers to the base class.

Reply:
agree that it's not needed